crates/sui-core/src/authority.rs

```diff
@@ -2852,17 +2851,17 @@ impl AuthorityState {
     /// compatible with the current versions of those packages on-chain.
     pub async fn get_available_system_packages(&self) -> Vec<ObjectRef> {
         let Some(move_stdlib) = self.compare_system_package(
-            MOVE_STDLIB_OBJECT_ID,
-            sui_framework::get_move_stdlib(),
-            sui_framework::get_move_stdlib_transitive_dependencies(),
+            MoveStdlib::ID,
+            MoveStdlib::as_modules(),
+            MoveStdlib::transitive_dependencies(),
         ).await else {
             return vec![];
         };
 
         let Some(sui_framework) = self.compare_system_package(
-            SUI_FRAMEWORK_OBJECT_ID,
+            SuiFramework::ID,
             sui_framework_injection::get_modules(self.name),
```

Comment:
Clarification q: we do the `sui_framework_injection` trick here for sui framework but not move stdlib purely for testing purposes right? When not in simtests, this just returns the actual framework code in the `sui-framework` folder.

Reply:
That's correct!